universal-login-wallet/cypress/support/topupCommands.ts

```diff
@@ -0,0 +1,31 @@
+Cypress.Commands.add('goToTopUp', () => {
+  cy.get('.funds-topup').click();
+});
+
+Cypress.Commands.add('selectFiat', () => {
+  cy.get('#topup-btn-fiat').click();
+});
+
+Cypress.Commands.add('chooseLanguage', () => {
+  cy.get('.unilogin-component-top-up-radio-label').should('be.exist');
+  cy.get('.unilogin-component-top-up-radio-label');
+  cy.get('.unilogin-component-country-select-toggle');
+  cy.wait(250);
```

Comment:
Could we not use wait, please? 🙏

Reply:
Without cy.wait() it crashes because of the .unilogin-component-country-select-btn element is detached or removed from DOM while .click() process somehow.